--- open-vm-tools-2010.03.20-243334/configure~	2010-03-22 23:38:16.000000000 +0100
+++ open-vm-tools-2010.03.20-243334/configure	2010-04-08 08:19:20.807751468 +0200
@@ -23983,6 +23983,7 @@
 echo "${ECHO_T}$ac_cv_lib_dl_dlopen" >&6; }
 if test $ac_cv_lib_dl_dlopen = yes; then
   VIX_LIBADD="$VIX_LIBADD -ldl"
+  VMWARE_USER_LDADD="$VMWARE_USER_LDADD -ldl"
        LIBVMTOOLS_LIBADD="$LIBVMTOOLS_LIBADD -ldl"
 else
   { { echo "$as_me:$LINENO: error: dlopen was not found, but is required for open-vm-tools to function properly. Please contact your OS vendor." >&5
